flake.nix

```diff
@@ -0,0 +1,41 @@
+{
+  description = "Cubical Agda";
+
+  inputs.flake-utils.url = github:numtide/flake-utils;
```

Comment:
https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md

Reply:
I changed the syntax.